src/controllers/tripController.js

```diff
@@ -19,3 +20,29 @@ export const oneWayTripRequest = async (req, res) => {
     return respondWithWarning(res, statusCode.internalServerError, 'Internal Server Error');
   }
 };
+
+/**
+* A function to create a return trip
+* @param {object} req
+* @param {object} res
+* @returns {object} response
+*/
+export const returnTripRequest = async (req, res) => {
+  const { id } = req.auth;
+  const { returnDate, departureDate } = req.body;
+  const convertReturnDateToISO = new Date(returnDate).toISOString();
```

Comment:
`convertReturnDateToISO` sounds like an action word which should be for functions/method naming. `dateToISO ` is more appropriate for a variable holding a value.

Reply:
Ok, i will fix that